### PR TITLE
Fix error with localizedFields in object bricks

### PR DESCRIPTION
**Pull request by blankse**
image

### CODE REVIEW

**Comment by blankse** [comment]
It was also not possible to set the language argument:
`Unknown argument \"language\" on field \"xyz\" of type \"object_Xyz\".`

File: src/GraphQL/FieldHelper/DataObjectFieldHelper.php [hunk 1]
```diff
@@ -168,12 +168,12 @@ public function getFieldDefinitionFromKey($class, $key, &$container = null)
             $brickDefinition = Definition::getByKey($brickType);
 
             if ($brickDescriptor) {
-                $fieldDefinition = $brickDefinition->getFieldDefinition($brickKey);
```
Comment (blankse) [inline src/GraphQL/FieldHelper/DataObjectFieldHelper.php:171]: It seems the getFieldDefinition() method of the object brick definition also get localized field definitions. So the $container variable isn't set for localized fields.

**Comment by blankse** [comment]
I changed it also for the class field definitions because the getFieldDefinition method in 11.x loads also the localized fields: https://github.com/pimcore/pimcore/pull/14105

File: src/GraphQL/FieldHelper/DataObjectFieldHelper.php (hunk 1 above)
Comment (mcop1) [reply src/GraphQL/FieldHelper/DataObjectFieldHelper.php:171]: You are right, that´s why we unified the behavior for pimcore 11 (see your other comment). Thanks!

**Comment by mcop1** [comment]
As always, thank you 👍

**Pull request merged**